### PR TITLE
added another js file

### PR DESCRIPTION
**Pull request by jimblizzard**
test2.js was added. Want to get input from others

### CODE REVIEW

File: test2.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// copilot demos
+
+function getLowestNumberInArray(array) {
+    var lowestNumber = array[0];
+    for (var i = 1; i < array.length; i++) {
+        if (array[i] < lowestNumber) {
+            lowestNumber = array[i];
```
Comment (jimblizzard) [inline test2.js:7]: Does this make sense to you?

**Review by jimblizzard — COMMENTED** [review]
this looks ok, right?

File: test2.js (hunk 1 above)
Comment (jimblizzardmsft) [reply test2.js:7]: yes it does.

**Review by jimblizzardmsft — COMMENTED** [review]
Looks good

File: test2.js (hunk 1 above)
Comment (jimblizzard) [reply test2.js:7]: Good to go then....

**Comment by jimblizzardmsft** [comment]
Good to go.

**Pull request merged**